<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialDialog</class>
 <widget class="QDialog" name="SerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial port settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>256</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Serial Port</string>
   </property>
   <widget class="QComboBox" name="cbSerialPortInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_portinfo">
     <item>
      <widget class="QLabel" name="labelDesc">
       <property name="text">
        <string>Description: N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelManu">
       <property name="text">
        <string>Manufacturer: N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSerNum">
       <property name="text">
        <string>Serial number: N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelLocation">
       <property name="text">
        <string>Location: N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelVendor">
       <property name="text">
        <string>Vendor Identifier: N/A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProd">
       <property name="text">
        <string>Product Identifier: N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Parameters</string>
   </property>
   <widget class="QWidget" name="layoutWidget_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>62</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_params">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Baudrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Data bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Stop bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Flow Control</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>81</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_values">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QComboBox" name="cbBaud">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDataBits">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbParity">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbStopBits">
       <property name="maxVisibleItems">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFlowControl">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QCheckBox" name="localEchoCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Local echo</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SerialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SerialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
